examples/kosmos-2-patch14-224/tests/test_kosmos_2_patch14_224.py

```diff
@@ -0,0 +1,40 @@
+from nos.client import Client
+from typing import List
+
+from rich import print
+from PIL import Image
+import requests
+
+TEST_PROMPTS = ["<grounding>An image of"]
+
+def test_kosmos_2_patch_224():
+    # Create a client
+    client = Client("[::]:50051")
+    assert client is not None
+    assert client.WaitForServer()
+    assert client.IsHealthy()
+
+    model_id = "microsoft/kosmos-2-patch14-224"
+    
+
+    model = client.Module(model_id)
+    assert model is not None
+    assert model.GetModelInfo() is not None
+    print(f"Test [model={model_id}]")
+
+    prompt = "<grounding>An image of"
+
+    url = "https://huggingface.co/microsoft/kosmos-2-patch14-224/resolve/main/snowman.png"
```

Comment:
I think we can skip downloading it since the image is attached, and remove `import requests`.

Reply:
There is a weird quirk of this model that the maintainers rocommend saving all images to ensure the outputs are consistent:     
`# The original Kosmos-2 demo saves the image first then reload it. For some images, this will give slightly different image input and change the generation outputs.`

We could skip this download step, and I'm happy to. However, the tests are kind of acting as usage examples. This inconsistency could really trip up a user in the future once they start using this model. In lieu of the external usages (a notebook, etc.) of this model being posted I would recommend keeping it as is and deleting the image to save space (which I've done for now). Happy to do as you requested as well, just LMK

Reply:
Ok, let's keep it like this.